### PR TITLE
Add description metadata for all files in Layouts-Modules, some content updates

### PR DESCRIPTION
**Pull request by nancyhild**
1) Added description metadata for all files in Layouts/Modules in BB. Updated content in some of these articles.
2) Deleted an outdated article about Text Editor module style changes. This made the Text Editor module the only article in that category, so moved it out of the category. Changed all cross-links from other files, removed category from sidebars.js, added redirects for all changes to netlify.toml. I didn't change yarn.lock, my understanding is that that file is not committed.

### CODE REVIEW

**Comment by nancyhild** [comment]
I don't have permissions to view the Netlify logs so I can't troubleshoot that. I did add some redirects to netlify.toml, perhaps that was a bad thing.

File: docs/beaver-builder/layouts/modules/audio.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 id: audio
 title: Audio
 sidebar_label: Audio
+description: The Audio module displays a an audio player with controls to play audio from the Media Library or an audio file from an external source.
```
Comment (Dannyholt) [inline docs/beaver-builder/layouts/modules/audio.md:5]:
```suggestion
description: The Audio module displays an audio player with controls to play audio from the Media Library or an audio file from an external source.
```

File: docs/beaver-builder/layouts/modules/button-group.md [hunk 2]
```diff
@@ -2,9 +2,10 @@
 id: button-group
 title: Button Group
 sidebar_label: Button Group
+description: The Button Group module displays display multiple buttons in a single column, either horizontally or vertically.
```
Comment (Dannyholt) [inline docs/beaver-builder/layouts/modules/button-group.md:5]:
```suggestion
description: The Button Group module displays multiple buttons in a single column, either horizontally or vertically. 
```

File: docs/beaver-builder/layouts/modules/button/make-a-button-transparent.md [hunk 3]
```diff
@@ -2,6 +2,7 @@
 id: make-a-button-transparent
 title: Make a button transparent
 sidebar_label: Make a button transparent
+description: Here's how to make a button's background color  semitransparent or completely transparent.
```
Comment (Dannyholt) [inline docs/beaver-builder/layouts/modules/button/make-a-button-transparent.md:5]:
```suggestion
description: Here's how to make a button's background color semitransparent or completely transparent. 
```

File: docs/beaver-builder/layouts/modules/icon-and-icon-group.md [hunk 4]
```diff
@@ -2,9 +2,12 @@
 id: icon-and-icon-group
 title: Icon and Icon Group
 sidebar_label: Icon and Icon Group
+description: The Icon and Icon Group modules displaya single icon plus optional text or a group of iconsl
```
Comment (Dannyholt) [inline docs/beaver-builder/layouts/modules/icon-and-icon-group.md:5]:
```suggestion
description: The Icon and Icon Group modules displaya single icon plus optional text or a group of icons
```

File: docs/beaver-builder/layouts/modules/login-form.md [hunk 5]
```diff
@@ -2,6 +2,7 @@
 id: login-form
 title: Login Form module
 sidebar_label: Login Form
+description: The Login Form module adds a login form to your layout. This is useful to create restricted arears of the site, such as member-only pages.
```
Comment (Dannyholt) [inline docs/beaver-builder/layouts/modules/login-form.md:5]:
```suggestion
description: The Login Form module adds a login form to your layout. This is useful to create restricted areas of the site, such as member-only pages.
```

File: docs/beaver-builder/layouts/modules/module-overview.md [hunk 6]
```diff
@@ -2,6 +2,7 @@
 id: module-overview
 title: Module Overview
 sidebar_label: Module Overview
+description: Beaver  Buildder includes 30 content modules that form the building blocks of adding content to your layouts.
```
Comment (Dannyholt) [inline docs/beaver-builder/layouts/modules/module-overview.md:5]:
```suggestion
description: Beaver Builder includes 30 content modules that form the building blocks of adding content to your layouts.
```

File: docs/beaver-builder/layouts/modules/posts/posts.md [hunk 7]
```diff
@@ -2,6 +2,7 @@
 id: posts
 title: Posts
 sidebar_label: Posts module
+description: The Posts module displays a set of posts or pages with various layouts, styles, and  content filters.
```
Comment (Dannyholt) [inline docs/beaver-builder/layouts/modules/posts/posts.md:5]:
```suggestion
description: The Posts module displays a set of posts or pages with various layouts, styles, and content filters.
```

**Comment by nancyhild** [comment]
Eegad! Thanks for catching all those that's embarrassing.

**Review by Dannyholt — APPROVED** [review]

**Pull request merged**